### DISCUSSION TITLE
Harpy Pro Version?

### DISCUSSION THREAD

**Discussion by edem84** [robertodoering/harpy#444] [General]
Hello, I love this Twitter client. 
When can I buy the pro version?
Thanks for this great job.

**Comment by robertodoering** [comment]
Thanks @edem84! I plan for a release this October or November.

**Comment by edem84** [comment]
Nice!!!! hope it will be an annual payment!
-------- Message d'origine --------De : Roberto Doering ***@***.***> Date : 02/10/2021  12:42  (GMT+00:00) À : robertodoering/harpy ***@***.***> Cc : edem84 ***@***.***>, Mention ***@***.***> Objet : Re: [robertodoering/harpy] Harpy Pro Version? (Discussion #444) 
Thanks @edem84! I plan for a release this October or November.

**Reply by robertodoering** [reply]
It will be a one time payment, no subscription needed.

**Comment by edem84** [comment]
Nice i love this
-------- Message d'origine --------De : Roberto Doering ***@***.***> Date : 02/10/2021  12:48  (GMT+00:00) À : robertodoering/harpy ***@***.***> Cc : edem84 ***@***.***>, Mention ***@***.***> Objet : Re: [robertodoering/harpy] Harpy Pro Version? (Discussion #444) 
It will be a one time payment, no subscription needed.

**Comment by ghost** [comment]
Currently I'm not able to use Google Play on my devices, is there any other option to get buy the application?

**Reply by robertodoering** [reply]
Unfortunately there are no other plans to support other platforms than the Play Store for Android and eventually the Appstore for iOS for now.

**Reply by Luke-L** [reply]
See, if you have a purchase option through your website (or just outside of the app store) you don't have to give money to those store platforms. 
Also love to hear about the one time payment instead of subscription :))

**Reply by bakinazik** [reply]
or you can create purchase key like in 2011. for the privacy lover..

**Comment by Edodaddo** [comment]
I would like to buy Harpy Pro on the Play Store to support this fantastic project but I have two questions:
- with the Pro version I would still have a limited number of likes that I can put (often with the free version of Harpy the message "Rate limit reached" appears)
- for those who buy Harpy Pro on the Play store, will there be any discounts for the purchase of Harpy Pro on iOS (when it comes out)?

Thanks and congratulations for this fantastic project.

**Reply by robertodoering** [reply]
>with the Pro version I would still have a limited number of likes that I can put (often with the free version of Harpy the message "Rate limit reached" appears)

The rate limits for likes are shared across every user, so in the free version you will hit the limit more frequently. Since the pro version has less users you are a lot less likely to hit the rate limit there, but there is no guarantee that it will stay that way and that you won't hit it at all unfortunately.

> for those who buy Harpy Pro on the Play store, will there be any discounts for the purchase of Harpy Pro on iOS (when it comes out)?

It would be difficult to automatically give people a discount but if you hit me up with an email an are able to proof that you purchases the android version I would definitely give you a discount.

**Comment by dnestico** [comment]
There is really nothing that he can do to avoid the rate limits, those are imposed by Twitter themselves. Depending how much you use the app it shouldn't be a big issue.

**Reply by Edodaddo** [reply]
Thanks for the info, I thought so but wasn't sure. It will mean that I will put fewer likes, to use a beautiful app like Harpy it is really worth it.

**Comment by Oyetomi** [comment]
Just a suggestion , you think it could be possible to make requests from
secondary accounts and pass it down to the main twitter account.

**Comment by VipedL** [comment]
Hmm how about giving users possibility to use their own apikeys?

**Reply by dnestico** [reply]
He has! https://github.com/robertodoering/harpy/releases/tag/0.9.8

**Reply by VipedL** [reply]
That's great. Now how do I do that? Can't seem to find a way in the app.

**Reply by VipedL** [reply]
Never mind, found it, I needed to log out for some reason.

**Reply by dnestico** [reply]
Yes that is correct you do need to log out first.

**Reply by KrychuTM** [reply]
Where did you find how to do it?

**Reply by robertodoering** [reply]
@KrychuTM
Tap on the top right button in the login screen to set your own api key

**Comment by killerlearner** [comment]
Can anybody tell me difference between pro version and free version

**Reply by robertodoering** [reply]
<img width=350 src=https://user-images.githubusercontent.com/20045287/194701604-bac3e87c-b1c2-4da4-bb08-be7f22ebd3e9.png />

**Comment by rei-zero246** [comment]
Would it be possible to contribute and receive a Play Store redeem code for those who are de-Googled?

**Reply by robertodoering** [reply]
I don't want people to contribute with the intention of just getting a pro key out of it, but if you do contribute in a valuable way I'm not against it.
That said, how are you going to redeem the code if you are "de-googled"? Wouldn't that imply you don't use the play store?

**Reply by rei-zero246** [reply]
I have a Google account with which I can redeem the code and then I can access it once it has been attached to the account. Let me know if you are willing and next steps.
